ui/src/components/pages/Pipeline/partials/Graph/partials/Spec/index.tsx

```diff
@@ -9,65 +9,50 @@ import {
   TableRow,
   Tabs,
 } from "@mui/material";
-import { SyntheticEvent, useState } from "react";
-import TabPanel from "../../common/Tab-Panel";
-import { a11yProps } from "../../../utils";
-import { Pipeline } from "../../../types/declarations/pipeline";
-
-interface SpecProps {
-  pipeline: Pipeline;
-}
+import TabPanel from "../../../../../../common/Tab-Panel";
+import { a11yProps } from "../../../../../../../utils";
+import { SpecProps } from "../../../../../../../types/declarations/graph";
 
 export default function Spec(props: SpecProps) {
   const { pipeline } = props;
 
-  const [value, setValue] = useState(0);
-
-  const handleChange = (event: SyntheticEvent, newValue: number) => {
-    setValue(newValue);
-  };
+  const fontWeightStyle = { fontWeight: "bold" };
 
   return (
     <Box>
       <Box sx={{ borderBottom: 1, borderColor: "divider" }}>
-        <Tabs
-          value={value}
-          onChange={handleChange}
-          aria-label="pipeline details"
-        >
+        <Tabs value={0} aria-label="pipeline details">
           <Tab
-            sx={{
-              fontWeight: "bold",
-              color: "grey",
-              fontFamily: "IBM Plex Sans",
-            }}
+            sx={fontWeightStyle}
             label="Pipeline Details"
             {...a11yProps(0)}
           />
         </Tabs>
       </Box>
-      <TabPanel value={value} index={0}>
+      <TabPanel value={0} index={0}>
```

Comment:
is this value always 0 ?

Reply:
we have a constant single tab here, thus setting it explicitly